packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1466,11 +1478,126 @@ export class ContainerRuntime extends EventEmitter
                 builder.addWithStats(key, contextSummary);
             }));
 
+        const deletedDataStores = this.runGarbageCollection();
+        builder.addBlob(".deletedDataStores", JSON.stringify(deletedDataStores));
```

Comment:
We should figure out proper mark up here, not put it into the blob. Storage is not going to read blobs.
Also any solution here should assume deletes on any level in the tree. Consider future of data stores being on channels, i.e. data store being a child of data store - the design that we put in place here should work with minimal differences for such case (DDSs is a another case that we need to make sure can be supported in future)

Reply:
Yeah, I added it here for now to make my prototpye testable :-).
I spoke to Arin at length about where / how we should put this information. We came up with a couple of ideas and I am going to play around with them in my next prototype.